Patch-Source: https://github.com/mapeditor/tiled/pull/4198/commits/60388da00ef8e64608c581369b5c68e85162e31d
---
From 60388da00ef8e64608c581369b5c68e85162e31d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Thorbj=C3=B8rn=20Lindeijer?= <bjorn@lindeijer.nl>
Date: Fri, 14 Feb 2025 20:49:16 +0100
Subject: [PATCH] Fixed compile against Qt 6.8.2

A rather smaller change than 776d3b67b011f970a65e8a743795401851684cc9
for the 1.11 branch, which keeps compatibility down to Qt 5.12.
---
 src/libtiled/qtcompat_p.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libtiled/qtcompat_p.h b/src/libtiled/qtcompat_p.h
index 3b46dc1be7..60c089ed7b 100644
--- a/src/libtiled/qtcompat_p.h
+++ b/src/libtiled/qtcompat_p.h
@@ -28,5 +28,5 @@ using ::endl;
 #endif
 
 #if QT_VERSION >= QT_VERSION_CHECK(6,0,0)
-using QStringRef = QStringView;
+#define QStringRef QStringView
 #endif
